src/include/sof/trace/trace.h

```diff
@@ -233,19 +233,26 @@ void _log_sofdict(log_func_t sofdict_logf, bool atomic, const void *log_entry,
 
 #ifdef CONFIG_LIBRARY
 
-extern int test_bench_trace;
+#include <sys/time.h>
+
+/* trace level used on host configurations */
+extern int host_trace_level;
+
 char *get_trace_class(uint32_t trace_class);
 #define _log_message(ignored_log_func, atomic, level, comp_class, ctx, id_1, id_2, format, ...)	\
-do {											\
-	(void)ctx;									\
-	(void)id_1;									\
-	(void)id_2;									\
-	if (test_bench_trace) {								\
-		char *msg = "(%s:%d) " format;						\
-		fprintf(stderr, msg, strrchr(__FILE__, '/') + 1,\
-			__LINE__, ##__VA_ARGS__);	\
-		fprintf(stderr, "\n");							\
-	}										\
+do {								\
+	(void)ctx;						\
+	(void)id_1;						\
+	(void)id_2;						\
+	struct timeval tv;					\
+	char *msg = "(%s:%d) " format;				\
```

Comment:
Why not put both these variables under the `if`? In practical terms the difference isn't huge - also this way their scope is limited to the "pseudo-loop," but having them under the `if` would be more logical